### PR TITLE
feat: SSH Keypair module via deployment script

### PR DESCRIPTION
**Pull request by vlahane**
## Description
Closes: https://github.com/Azure/bicep-registry-modules/issues/384

If you haven't already, read the full [contribution guide](https://github.com/Azure/bicep-registry-modules/blob/main/CONTRIBUTING.md). The guide may have changed since the last time you read it, so please double-check. Once you are done and ready to submit your PR, edit the PR description and run through the relevant checklist below.

Enable GitHub Worksflows in your fork to enable auto-generation of assets with our [GitHub Action](/.github/workflows/push-auto-generate.yml).
To trigger GitHub Actions after auto-generation, [add a GitHub PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) as a secret in your forked repository called `PAT`.

## Adding a new module

- [ ] A proposal has been submitted and approved.
- [x] I have included "Closes #{module_proposal_issue_number}" in the PR description.
- [x] I have run `brm validate` locally to verify the module files.
- [x] I have run deployment tests locally to ensure the module is deployable.

### CODE REVIEW

**Comment by vlahane** [comment]
@Gordonby @shenglol  please review! thanks 😄

File: modules/deployment-scripts/create-kv-sshkeypair/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# SSH Key Pair Creation
+
+Creates SSH Key Pair Stores them in KeyVault as Secrets
+
+## Description
+
+Creates SSH key pair and stores them in provided AKV.
+
+{{ Add detailed description for the module. }}
```
Comment (dciborow) [inline modules/deployment-scripts/create-kv-sshkeypair/README.md:9]:
```suggestion
Creates SSH key pair and stores them in provided AKV.
```

**Review by dciborow — APPROVED** [review]

**Comment by vlahane** [comment]
@dciborow @shenglol  module was not approved and mcr onboarded, will you please do! thanks!

**Comment by Gordonby** [comment]
> @Gordonby @shenglol please review! thanks 😄

Most confusing PR ever...

I saw it get merged and was like "did i do that"? 😆 
Then i saw i didn't author it, but the code looked really familiar 🤣

Great work. 👍

**Comment by shenglol** [comment]
Well, we need to onboard the MCR manifest for it before merging it. I'll publish the module manually once it's done.

**Pull request merged**